spec/features/loading_websites_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.feature "Loading Website", type: :feature do
+  it "displays page title" do
+    visit "/todos"
+
+    expect(page).to have_css("header h1", text: "todos")
+  end
```

Comment:
This is better served a  view spec. You can learn more about when to use which type of spec [through this article](https://robots.thoughtbot.com/how-we-test-rails-applications). (I read it once a week).

In general, I would say this isn't really a feature spec. It lacks the exercise component of a [4-phase test](https://robots.thoughtbot.com/four-phase-test). Feature specs should build off of some user action. For instance:
```
visit root_path

click_on "todos"

expect(page).to have_css(...)
```

In that example, you'd be testing that the link actually points to the expected destination. I'd say that too is a bit too light for a feature spec though.

Also, great use of the `text` attribute with the `have_css` matcher. I forget about it from time to time and I think learning to use it right away will help you write more fluid and expressive tests.